helm/alfresco-content-services/templates/secret-database.yaml

```diff
@@ -1,10 +1,10 @@
 apiVersion: v1
 kind: Secret
 metadata:
-  name: {{ template "content-services.shortname" . }}-dbsecret
+  name: {{ .Values.global.repo.databaseSecretName }}
   labels:
     {{- include "repository.labels" . | nindent 4 }}
 type: Opaque
 data:
-  DATABASE_USERNAME: {{ .Values.database.user | default .Values.postgresql.postgresqlUsername | b64enc | quote }}
-  DATABASE_PASSWORD: {{ .Values.database.password | default .Values.postgresql.postgresqlPassword | b64enc | quote }}
+  {{ .Values.global.repo.databaseSecretUsernameKey }}: {{ .Values.database.user | default .Values.postgresql.postgresqlUsername | b64enc | quote }}
+  {{ .Values.global.repo.databaseSecretPasswordKey }}: {{ .Values.database.password | default .Values.postgresql.postgresqlPassword | b64enc | quote }}
```

Comment:
We can't leave the option to override the pg credentials in the `database` context otherwise there's no way for the subchart to authenticate to the db.
Also I don't see the benefit of using input values as keys neither for secret name. It just makes things a lot more complex to configure.
I'd rather we instead use static values and reference those in the subchart

Reply:
those values are defined to avoid repeating static strings here and in the subchart, but I would argue that for the user having Values that are not meant to be changed are just confusing?

I can just use `.Release.name` in metadata.name here and in the subchart, and DATABASE_* as static strings

Reply:
Can't we proceed as if the subchart is primarily meant to be used with the acs-deployment chart (and so use  "content-services.shortname" to expand the secret name just like in the main chart) and default to something else in case that template is not avaialble (e.g. an input value which would only live in the subchart values)?